### PR TITLE
Feat - added alert and button component

### PR DESCRIPTION
**Pull request by Mohasin-Haque**
image
Alert Boxes
image
Buttons

### CODE REVIEW

**Review by saifudinbadani — APPROVED** [review]
looks good to merge.

File: pages/alert.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            <div class="right-nav">
+                <ul class="list">
+                    <li><a href="../index.html" class="nav nav-pills">Home</a></li>
+                    <li><a href="./avatar.html" class="nav right-nav-active nav-pills">Documentation</a></li>
+                    <li><a href="./installation.html" class="nav nav-pills">Installation</a></li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+    <div class="main-container">
```
Comment (amirnathani9) [inline pages/alert.html:30]: give the proper class name

File: pages/alert.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            <p class="subtext">Components</p>
+            <a href="./avatar.html" class="item">Avatar</a>
+            <a href="./alert.html" class="item">Alert</a>
+            <a href="./badge.html" class="item">Badges</a>
+            <a href="./button.html" class="item">Button</a>
+            <a href="./card.html" class="item">Card</a>
+            <a href="./dialogue.html" class="item">Dialogue</a>
+            <a href="./image.html" class="item">Image</a>
+            <a href="./input.html" class="item">Input</a>
+        </div>
```
Comment (amirnathani9) [inline pages/alert.html:41]: give the proper name to the class

File: pages/alert.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        <iframe class="code-snippet-display"
+            src="https://carbon.now.sh/embed?bg=rgba%28255%2C59%2C48%2C1%29&t=base16-light&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=82px&dsblur=100px&wc=true&wa=true&pv=27px&ph=22px&ln=false&fl=1&fm=IBM+Plex+Mono&fs=14px&lh=143%25&si=false&es=2x&wm=false&code=%2520%2520%253C%21--%2520Alert%2520Varients%2520--%253E%2520%2520%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522alert%2520alert-success%2522%253EA%2520simple%2520success%2520alert.%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522alert%2520alert-danger%2522%253EA%2520simple%2520danger%2520alert.%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522alert%2520alert-warning%2522%253EA%2520simple%2520warning%2520alert.%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522alert%2520alert-info%2522%253EA%2520simple%2520info%2520alert.%253C%252Fdiv%253E"
+            sandbox="allow-scripts allow-same-origin">
```
Comment (amirnathani9) [inline pages/alert.html]: format the code properly

File: pages/avatar.html [hunk 4]
```diff
@@ -74,11 +76,11 @@ <h3 class="heading heading-avatar">Same size avatar</h3>
         <h3 class="heading heading-avatar">Avatar with status</h3>
         <p class="description description-avatar">To show the status of avatar, you need to include class <span
                 class="author">avatar</span> for avatar in img tag and class <span class="author">status</span> for
-                status in saprate add class according to status <span class="author">status-online, status-busy, status-dnd,
+            status in saprate add class according to status <span class="author">status-online, status-busy, status-dnd,
```
Comment (amirnathani9) [inline pages/avatar.html]: do the format properly

File: pages/button.html [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+            src="https://carbon.now.sh/embed?bg=rgba%28255%2C59%2C48%2C1%29&t=base16-light&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=82px&dsblur=100px&wc=true&wa=true&pv=27px&ph=22px&ln=false&fl=1&fm=IBM+Plex+Mono&fs=14px&lh=143%25&si=false&es=2x&wm=false&code=%253C%21--%2520Primary%2520Button%2520%28%2520Outline%2520%29--%253E%250A%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522btn%2520btn-primary%2522%253EOutlined%2520Primary%253C%252Fbutton%253E%250A"
+            sandbox="allow-scripts allow-same-origin">
+        </iframe>
+        <div class="flex-center alert-center">
+            <button class="btn btn-primary">Outlined Primary</button>
+        </div>
+        <!-- Solid primary buttom -->
+        <h3 class="heading heading-avatar">Primary Button (Solid)</h3>
+        <p class="description description-avatar">Primary button is the main button which performs primary actions of
+            your app. You can use <span class="author">btn</span> for button and for particular button use <span
```
Comment (amirnathani9) [inline pages/button.html]: give the proper class name

File: styles/alert.css [hunk 6]
```diff
@@ -0,0 +1,26 @@
+.alert {
+  width: 50%;
+  padding: 1rem;
+  margin: 0.5rem;
+  border-radius: 10px;
+}
+
+.alert-success {
+  background-color: var(--status-online);
+  color: var(--dark-white);
```
Comment (amirnathani9) [inline styles/alert.css]: dont use the color name instead use primary ,secondary,etc

File: styles/alert.css [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+.alert-warning {
+  background-color: var(--status-idle);
+  color: var(--dark-white);
+}
+
+.alert-info {
+  background-color: var(--item-background-color);
+  color: var(--dark-white);
+}
```
Comment (amirnathani9) [inline styles/alert.css]: same changes of color

**Comment by amirnathani9** [comment]
all goods good the ui and all. do the changes as suggested

File: styles/alert.css (hunk 7 above)
Comment (Mohasin-Haque) [reply styles/alert.css]: Done!!

File: styles/alert.css (hunk 6 above)
Comment (Mohasin-Haque) [reply styles/alert.css]: Done!!

File: pages/alert.html (hunk 1 above)
Comment (Mohasin-Haque) [reply pages/alert.html:30]: done!!

**Pull request merged**